### PR TITLE
internal: change shared library build destination

### PR DESCRIPTION
**Pull request by Korisss**
Without this fix, I get a ChakraCore linking error (CMake can't find the ChakraCore library) and function not found error in ScriptStorage.cpp

### CODE REVIEW

**Comment by Korisss** [comment]
изображение

**Comment by Korisss** [comment]
изображение
изображение

**Comment by Pospelove** [comment]
@nic11

File: skymp5-server/cpp/server_guest_lib/ScriptStorage.cpp [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #include "ScriptStorage.h"
+#include <cstring>
```
Comment (Wedmer) [inline skymp5-server/cpp/server_guest_lib/ScriptStorage.cpp:2]: How this change relates to ChakraCore linking problem?

**Review by Wedmer — CHANGES_REQUESTED** [review]
Everything is ok except unrelated change in skymp5-server includes.

**Comment by Pospelove** [comment]
https://github.com/skyrim-multiplayer/skymp/blob/main/docs/docs_maintainer_rules.md#commit-names

**Comment by nic11** [comment]
Thanks for the PR, it was upstreamed to vcpkg (https://github.com/microsoft/vcpkg/pull/28689) and will be applied with vcpkg update: #1295

Also, looks fine for my build now :+1: 
Screenshot_20230105_162806

**Pull request closed without merging**